doc/api/fs.md

```diff
@@ -4126,6 +4126,10 @@ Asynchronous stat(2). The callback gets two arguments `(err, stats)` where
 
 In case of an error, the `err.code` will be one of [Common System Errors][].
 
+[`fs.stat()`][] follows symbolic links, [`fs.lstat()`][] looks at the links
+themselves. Therefore, it is recommended to use [`fs.lstat()`][] when the links
+themselves need to be examined.
```

Comment:
```suggestion
themselves. Therefore, [`fs.lstat()`][] must be used when the links themselves
need to be examined.
```
because it's not optional, it's a hard requirement